### DISCUSSION TITLE
How should I resolve the scrollarea error in recent macos webkit ?

### DISCUSSION THREAD

**Discussion by olup** [radix-ui/primitives#680] [Help] [answered]
I am getting this same error : https://github.com/radix-ui/primitives/issues/500#issuecomment-816308604

The thread is not explicit about how it should be resolved. The final merge request hints that the fix has been merged - however I am still getting an error. Is there a polyfill or something I should add ?

**Comment by olup** [comment, marked as answer]
Even browsing the scrollArea docs from safari (macos Catalina running in KVM) just displays an error. So I guess it's just plain not suported.

**Reply by benoitgrelard** [reply]
We have merged a new version of the `ScrollArea` in our `main` branch but it hasn't been released yet. It fixes this issue. We should be able to release it soon.

**Reply by olup** [reply]
ow ok nice ! I could `yarn` it from github then I guess

**Reply by benoitgrelard** [reply]
Hey, this was now released.
https://radix-ui.com/primitives/docs/components/scroll-area
